### PR TITLE
[SofaKernel] Move ScriptEvent class from SofaPython to core/objectModel

### PR DESCRIPTION
**Pull request by epernod**
Like the port salut, everything is in the title. 

______________________________________________________

This PR: 
- [ ] builds with SUCCESS for all platforms on the CI.
- [ ] does not generate new warnings.
- [ ] does not generate new unit test failures.
- [ ] does not generate new scene test failures.
- [ ] does not break API compatibility.
- [ ] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by guparan** [comment]
Ok but why ?

**Comment by epernod** [comment]
just because it is useful in the rest of Sofa in cpp without having sofaPython. 
If you have a better place in /modules it is ok for me. I looked but I saw all events where stored in objectModel

File: applications/plugins/SofaPython/PythonScriptEvent.h [hunk 1]
```diff
@@ -25,7 +25,7 @@
 #include "PythonCommon.h"
 
 #include <SofaPython/config.h>
```
Comment (marques-bruno) [inline applications/plugins/SofaPython/PythonScriptEvent.h]: Are those still necessary?

**Review by marques-bruno — APPROVED** [review]
RAS, clean PR

**Pull request merged**